### ISSUE TITLE
Nvidia 495 & GBM

### ISSUE THREAD

**Issue by Rosentti** [mpsq/arewewaylandyet#113]
As we all know, nvidia has implemented GBM in their 495 beta series driver. This does work for most purposes (Gnome, Sway, etc), however with the big caveat of KDE Plasma not working. So the Nvidia portion should be added say something about these beta drivers.

**Comment by svenstaro** [comment]
I'd suggest to wait until there's a stable release of those drivers.

**Comment by 20PercentRendered** [comment]
> > (Gnome, Sway, etc), however with the big caveat of KDE Plasma not working.
> 
> Support in KWin was merged a week ago: https://invent.kde.org/plasma/kwin/-/merge_requests/1526

Well, support in KWin. However, qt-wayland is still affected by a [bug](https://codereview.qt-project.org/c/qt/qtwayland/+/373473) that makes qt applications running on Nvidia GBM be unusably slow.

**Comment by rstacruz** [comment]
A stable release of NVIDIA 495 is now out, and Sway can work with the `--unsupported-gpu` flag.

**Comment by svenstaro** [comment]
Yeah, I suggest the list be updated now to reflect the new state of affairs.

**Comment by ilmari-lauhakangas** [comment]
Unfortunately the same as on Nov 14, 2021 - just click the link and you will see the fix to Qt is still not merged.

**Comment by ilmari-lauhakangas** [comment]
David Edmundson commented in the Qt patch [Don't block in QWaylandDisplay::flushRequests](https://codereview.qt-project.org/c/qt/qtwayland/+/373473)

> This particular issue is solved via the other now merged request with the new event thread.

He didn't give a link, but I assume he is referring to his patch from March called [client: Avoid trying to read/cancel read if wl_display_read_events failed](https://codereview.qt-project.org/c/qt/qtwayland/+/394765) as both patches are touching src/client/qwaylanddisplay.cpp and deal with the topic of wl_display_read_events
